src/libutil/filter.cpp

```diff
@@ -289,8 +281,20 @@ class FilterBlackmanHarris1D : public Filter1D {
         const float A2 =  0.14128f;
         const float A3 = -0.01168f;
         const float m_pi = float (M_PI);
+#if 0
```

Comment:
Yeah the functions are maybe poorly named `fast_sinpi` shoould really be `veryfast_sinpi`. But `fast_sin` and `fast_cos` are close to full precision.